src/olympia/legacy_api/templates/legacy_api/includes/addon.xml

```diff
@@ -26,8 +26,8 @@
     {%- endfor %}
     {% endif %}
   </authors>
-  <summary>{{ addon.summary|strip_html(new_api) }}</summary>
-  <description>{{ addon.description|strip_html(new_api) }}</description>
+  <summary>{%- if new_api -%}{{ addon.summary }}{% else %}{{ addon.summary|clean(true) }}{% endif -%}</summary>
```

Comment:
this is pretty old, see above: `{% set new_api = (api_version >= 1.5) %}`. Not sure if any clients are using a version older than 1.5 (which is the current version for the legacy API)...